src/pages/logos/LogoDeepSearch.jsx

```diff
@@ -88,6 +89,18 @@ export default function LogoSearch() {
     };
   }, [searchState, searchCount]);
 
+  React.useEffect(() => {
+    if (selectAll) {
+      setLogosToAnnotate((prev) =>
+        prev.map((logo) => ({ ...logo, selected: true }))
+      );
```

Comment:
This suffers from one problem: pagination.

This select all will also select logos on the previous page and the next page that you do not see currently. It could work if restricted to the current page logos